### DISCUSSION TITLE
(Github Pages) No language(s) shown for repository due to docs/

### DISCUSSION THREAD

**Discussion by ArjunSharda** [community/community#19424] [Projects and Issues] [answered]
There is no language detected by github for my repository due to the folder it's in. This is a bug as overall it should show the language informations.

My repository: https://github.com/ArjunSharda/TimeConv

**Comment by melvincwng** [comment, marked as answer]
@ArjunSharda, for bugs, you can report them to [GitHub Bug Report](https://support.github.com/contact/bug-report).

Hope this helps.

Kindly 'Mark as answer' if this reply has been helpful/addressed your concerns.

Thank you & have a great day ahead 👍🏻
